buildprocess/configureWebpack.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+/*global require*/
```

Comment:
Ah, good point.  I may have turned off the rule that would yell at us for the superfluous global because I didn't want to change every source for in TerriaJS.  :)

In any case, no reason to add it to new source files!